plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -230,7 +230,7 @@ protected void verifyConcurrentUpdateFailurePermissible(Exception e)
     protected void verifyConcurrentAddColumnFailurePermissible(Exception e)
     {
         assertThat(e)
-                .hasMessageContaining("Cannot update Iceberg table: supplied previous location does not match current location");
+                .hasMessageContaining("Failed to replace table due to concurrent updates");
```

Comment:
What happened here?

Reply:
I've updated the check to check both the exception's message and the one from the underlying cause